### PR TITLE
New introduction

### PR DESCRIPTION
**Pull request by awholegunch**
First draft of new introduction. What's to come: 
- integrate output from running CDAP commands, as that was not included in source materials.
- additional commentary
- update current getting started in Developers' Manual to be an "introduction to development"
- Improve side-bar navigation and table of contents

Staged at: [New Introduction](http://stg-web101.sw.joyent.continuuity.net/cdap/3.0.0-SNAPSHOT-3.0_new_introduction/en/introduction/index.html)

Update: Moved the introduction into its own "manual" to ease navigation. Started to add console output, but the design of that is going to take some work.

### CODE REVIEW

File: cdap-docs/introduction/source/index.rst [hunk 1]
```diff
@@ -0,0 +1,1131 @@
...
+        |         |         |         |         |         | -0400   | .0      |         |         |         | ows NT 5.1)   |
+        +===================================================================================================================+
+
+       |non-breaking-space|
+
+.. list-table::
+   :widths: 15 65 20
+
+   * - **Current Approach**
+     - Write a code to compute the various stats: Unique, Histograms, etc.
```
Comment (jwang47) [inline cdap-docs/introduction/source/index.rst]: Write code

File: cdap-docs/introduction/source/index.rst [hunk 2]
```diff
@@ -0,0 +1,1131 @@
...
+     - HiveServer, Beeline
+
+.. list-table::
+   :widths: 15 85
+
+   * - **CDAP**
+     - ``$ get stream-stats logEventStream limit 1000``
```
Comment (jwang47) [inline cdap-docs/introduction/source/index.rst]: Not sure if `$` is good to use here. Usually I see that used a prefix for commands meant to executed in bash.

File: cdap-docs/introduction/source/index.rst [hunk 3]
```diff
@@ -0,0 +1,1131 @@
...
+     - - HBase
+       - HDFS
+
+.. list-table::
+   :widths: 15 85
+
+   * - **CDAP**
+     - Dataset that is time partitioned::
+     
+        $ list dataset instances
```
Comment (jwang47) [inline cdap-docs/introduction/source/index.rst]: This got rendered funny in the resulting html: `$ list dataset instances - Dataset that is time paritioned`

File: cdap-docs/introduction/source/index.rst [hunk 4]
```diff
@@ -0,0 +1,1131 @@
...
+     - - Check Oozie
+       - Check YARN Console
+     - - Oozie
+       - YARN
+
+.. list-table::
+   :widths: 15 85
+
+   * - **CDAP**
+     - ``$ describe app Wise``
```
Comment (jwang47) [inline cdap-docs/introduction/source/index.rst]: Not sure if this is a compelling example. All it does is list the programs that are a part of the app.

File: cdap-docs/introduction/source/index.rst [hunk 5]
```diff
@@ -0,0 +1,1131 @@
...
+.. list-table::
+   :widths: 45 45 10
+   :header-rows: 1
+
+   * - New Paradigm With CDAP
+     - Current Approach and Required Technologies
+     - 
+
+   * - CDAP
+     - - Bringing in different open source technologies that have different design principles
```
Comment (jwang47) [inline cdap-docs/introduction/source/index.rst]: Tense changed here.

File: cdap-docs/introduction/source/index.rst [hunk 6]
```diff
@@ -0,0 +1,1131 @@
...
+
+.. list-table::
+   :widths: 45 45 10
+   :header-rows: 1
+
+   * - New Paradigm With CDAP
+     - Current Approach and Required Technologies
+     - 
+
+   * - CDAP
```
Comment (jwang47) [inline cdap-docs/introduction/source/index.rst]: Probably want to list corresponding points in CDAP here. For example,

CDAP: Learn a single framework that works with multiple technologies
Without CDAP: Familiarize and learn how to operationalize the different technologies

File: cdap-docs/introduction/source/index.rst (hunk 1 above)
Comment (awholegunch) [inline cdap-docs/introduction/source/index.rst]: Fixed.

**Comment by awholegunch** [comment]
This PR needs to be revised:
- Once the adaptors are restored, it needs to have the CLI examples re-run for the adaptor example portion and add the (correct) output.
- The "Required Technologies" needs to be checked for each step.

**Comment by sreevatsanraman** [comment]
Remove the technologies required in the last table. Fix and merge.

**Pull request merged**